Rubberduck.CodeAnalysis/Inspections/Concrete/ProcedureNotUsedInspection.cs

```diff
@@ -14,32 +15,90 @@ namespace Rubberduck.CodeAnalysis.Inspections.Concrete
     /// </summary>
     /// <why>
     /// Unused procedures are dead code that should probably be removed. Note, a procedure may be effectively "not used" in code, but attached to some
-    /// Shape object in the host document: in such cases the inspection result should be ignored. An event handler procedure that isn't being
-    /// resolved as such, may also wrongly trigger this inspection.
+    /// Shape object in the host document: in such cases the inspection result should be ignored.
     /// </why>
     /// <remarks>
     /// Not all unused procedures can/should be removed: ignore any inspection results for 
     /// event handler procedures and interface members that Rubberduck isn't recognizing as such.
+    /// Public procedures of Standard Modules are not flagged by this inspection regardless of
+    /// the presence or absence of user code references.
     /// </remarks>
     /// <example hasResult="true">
-    /// <module name="MyModule" type="Standard Module">
+    /// <module name="Module1" type="Standard Module">
     /// <![CDATA[
     /// Option Explicit
     /// 
-    /// Public Sub DoSomething()
-    ///     ' macro is attached to a worksheet Shape.
+    /// Private Sub DoSomething()
     /// End Sub
     /// ]]>
     /// </module>
     /// </example>
     /// <example hasResult="false">
-    /// <module name="MyModule" type="Standard Module">
+    /// <module name="Module1" type="Standard Module">
     /// <![CDATA[
     /// Option Explicit
-    ///
+    /// 
     /// '@Ignore ProcedureNotUsed
+    /// Private Sub DoSomething()
+    /// End Sub
+    /// ]]>
+    /// </module>
+    /// </example>
+    /// <example hasResult="false">
+    /// <module name="Module1" type="Standard Module">
+    /// <![CDATA[
+    /// Option Explicit
+    /// 
+    /// Public Sub DoSomething()
+    /// End Sub
+    /// ]]>
+    /// </module>
+    /// </example>
+    /// <example hasResult="true">
+    /// <module name="Class1" type="Class Module">
+    /// <![CDATA[
+    /// Option Explicit
+    /// 
+    /// Public Sub DoSomething()
+    /// End Sub
+    /// 
+    /// Public Sub DoSomethingElse()
+    /// End Sub
+    /// ]]>
+    /// </module>
+    /// <module name="Module1" type="Standard Module">
+    /// <![CDATA[
+    /// Option Explicit
+    /// 
+    /// Public Sub ReferenceOneClass1Procedure()
+    ///     Dim target As Class1
+    ///     Set target = new Class1
+    ///     target.DoSomething
+    /// End Sub
+    /// ]]>
+    /// </module>
+    /// </example>
+    /// <example hasResult="false">
+    /// <module name="Class1" type="Class Module">
+    /// <![CDATA[
+    /// Option Explicit
+    /// 
     /// Public Sub DoSomething()
-    ///     ' macro is attached to a worksheet Shape.
```

Comment:
That was not redundant or contradicting the markup though? Sorry to sound like I'm dancing back and forth, but comments like this help clarify an example.

What's important is that things like module names and types, and example attributes ("hasresult"), are consistent with the code examples - wasn't the case before this commit, but now the pendulum went all the way to the other end....

Reply:
I looked at this a little differently...I did not delete the comment not because I thought it was contradictory to the markup - I agree, it is not.  Rather, I deleted it because I felt it falsely implied to our users that somehow RD 'knew' it was attached to a worksheet shape - and that's _why_ a result is not generated.  The real reason is as stated in the `<remarks>` - that any Public procedure in a Standard Module is ignored - the possibility of attachment to a worksheet shape is one example of why these types of procedures are ignored.

I'm happy to restore the comment - though I might suggest it read something like "macro may be attached to a worksheet Shape or other use"...unless I'm missing something about the original comment (which is also entirely possible).   

Just let me know if my modified version of the comment has merit or you would like it simply restored 'as it was' - I'm good either way.

Reply:
Good thinking, hadn't realized that implication, makes sense.

We can use module names too; if the module is named `Macros` (std module), that an convey useful information without being too wordy. And we can restate the condition in a comment, e.g. "`' public procedure in a standard module`

Reply:
Latest update/sync restores a comment within this example